api/src/main/kotlin/team/backend/presentation/quiz/QuizController.kt

```diff
@@ -15,16 +15,15 @@ import team.backend.response.SuccessResponse
 @RestController
 class QuizController(
     private val quizService: QuizService,
-
     private val quizMapper: QuizMapper,
 ) {
 
     @GetMapping("/v1/quiz/random")
-    suspend fun randomQuizzes(): String {
-        val result = RequestContext.getValue(Headers.AUTHORIZATION)
-        println("111Result: $result")
+    suspend fun random(): SuccessResponse<QuizDto.RandomResponse> {
+        val authorization = RequestContext.getValue(Headers.AUTHORIZATION)
```

Comment:
Isn't it better to get it from the domain layer (Service Class)?

Reply:
@mangchhe 
I think it is close to the role of the presentation layer to take out and use the value in the header.

Reply:
@BAEKJungHo 
All parts that require authority should be handed over as parameters.
Also, there is one value now, but it may be several in the future, so I think I should think about whether to do it in a way that I hand it over every time as parameter.

Reply:
@mangchhe Yes. You said about `authority`. 
Authority is the responsibility of the outer layer.
If there are more parameters, you can hand them over to the object.

Reply:
I think it is correct to process it in the same layer even if the header contains multiple values, just like json body contains multiple values.